nyxx_lavalink/lib/src/model/PlayParameters.dart

```diff
@@ -0,0 +1,75 @@
+part of nyxx_lavalink;
+
+/// Parameters to start playing a track
+class PlayParameters {
+  final Node _node;
+  /// The track to play
+  final Track track;
+  /// The guild where the track will be played
+  final Snowflake guildId;
+  /// Wether to replace the track or not
+  bool replace;
+  /// The time at where the track will start to play
+  Duration startTime;
+  /// The time at where the track will stop playing
+  Duration? endTime;
+
+  /// The requester of the track
+  Snowflake? requester;
+
+  /// The channel where this track was requested
+  Snowflake? channelId;
```

Comment:
missing final

Reply:
i want users to be able to manually modify this after enqueuing or playing the track, so missing final is not intended there